src/routes/Home.js

```diff
@@ -33,28 +78,44 @@ export default function Home() {
     navigate('/updates');
   }
 
+  const navigateLogin = () => {
+    navigate('/login');
+  }
+
+  const navigateSignup = () => {
+    navigate('/Signup');
+  }
+
   //TODO: Pages still needed: login, signup 
 
   return <>
     <div>{greeting}</div>
-    <button onClick = {navigateAboutUs}>
+    <Button onClick = {navigateAboutUs}>
```

Comment:
same thing here as what I said in my long comment in signup.js. u should just be able to do `onClick={() => navigate('Signup')}` directly instead of going to a middle man function

Reply:
Done